diskimage_builder/elements/ubuntugpu/install.d/51-gpu

```diff
@@ -73,4 +85,4 @@ libnvinfer-dev=5.1.5-1+cuda10.0
 
 # Install tensorflow-gpu 
 # --> There is a need to convert v1.X code to v2.X. See: https://www.tensorflow.org/guide/migrate
-pip install tensorflow-gpu
+pip install --upgrade tensorflow
```

Comment:
You've installed `python3-pip`, not sure `pip` (python3) is available?!?

`pip3 install --upgrade tensorflow`